### DISCUSSION TITLE
can foxglove add some examples of how to use updateParameterValues & publishParameterValues interfaces (c++) in a server

### DISCUSSION THREAD

**Discussion by xiao-yang25** [foxglove/community#989] [Q&A] [answered]
I create a server and receive parameters from a client, what i want to do is that when receive parameters, the server can send some backfeed to the client. i need a example of cpp.thanks.

**Comment by achim-k** [comment, marked as answer]
When parameters are set by the client, the server can send updated / new parameters to the client. This is for example done in the [foxglove_bridge server](https://github.com/foxglove/ros-foxglove-bridge/blob/e37a3e1e309c1a6255e3df5a86324e002dab65e1/ros1_foxglove_bridge/src/ros1_foxglove_bridge_nodelet.cpp#L692-L738) which is written in C++

**Reply by xiao-yang25** [reply]
Thanks. I used publishParameterValues（） ​​as an interface to publish responses from the server for clients. I just finished the first version of the code, so replying a little late.
